src/proxy_app/telemetry/logger.py

```
@@ -0,0 +1,267 @@
+        async def async_log_stream_event(self, *a, **kw): pass
+        def log_pre_api_call(self, *a, **kw): pass
+        def log_post_api_call(self, *a, **kw): pass
+
+log = logging.getLogger("telemetry")
+log.setLevel(logging.INFO)
+if not log.handlers:
+    h = logging.StreamHandler()
+    h.setFormatter(logging.Formatter("%(asctime)s [telemetry] %(levelname)s %(message)s"))
+    log.addHandler(h)
+
+DB_PATH = os.environ.get("TELEMETRY_DB_PATH", "/dev/shm/telemetry.db")
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/dev/shm/telemetry.db"

(S108)